dir-spec.txt

```diff
@@ -1588,6 +1588,14 @@
    and sending it in an HTTP POST request to each other authority at the URL
      http://<hostname>/tor/post/vote
 
+   If an authority has used a Bandwidth List file to generate this vote it
```

Comment:
Please don't put the bandwidth file URL in the middle of the vote section. It makes the rest of the vote section confusing.

You can put it before section 3.4.1., or in its own subsection 3.4.3.

Reply:
i better move all the paragraph (not just the URL), right?

Reply:
Yes, please move all the new content somewhere else.